core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/filter/SpecializedFilterOpExec.java

```diff
@@ -16,12 +18,20 @@ public SpecializedFilterOpExec(SpecializedFilterOpDesc opDesc) {
                 (Function1<Tuple, Boolean> & Serializable) this::filterFunc);
     }
 
+    // MARK: Debug/unittests only
+    public SpecializedFilterOpExec(SpecializedFilterOpDesc opDesc, List<FilterPredicate> predicates) {
```

Comment:
why do we need this function? You can create an opdesc with predicates in the test and pass it to the existing constructor in this file .. right?

Reply:
Sure, but then in the test file this same thing would be expanded to three lines involving assigning the public variable directly. But I made the fix anyways. Here's how it'll look:
```
var opDesc = new SpecializedFilterOpDesc()
opDesc.predicates = asList(
  new FilterPredicate("string", ComparisonType.IS_NULL, null)
)
val opExec = new SpecializedFilterOpExec(opDesc)
```